### PR TITLE
docs: update examples to be written by human not converter

### PR DESCRIPTION
**Pull request by derberg**
- adjusted all the channel names, messages names and operations names that were generated by converter to be more human friendly. My main concert was that having a generated channel name that is the same as `address` would bring confusion and suggest bad practice
- added if few places missing quotations - related https://github.com/asyncapi/converter-js/issues/197
- websocket kraken had few issues that needed a fix as they were kinda written still from a perspective of the user
- the other changes are commented inline

**What I did not do**
So social media example could be modified further imho. It was limited by v2 and could not show sharing of channels and operations. Now I think it should be refactored to do so too. Wdyt?

### CODE REVIEW

File: examples/gitter-streaming-asyncapi.yml [hunk 1]
```diff
@@ -148,7 +148,15 @@ components:
               description: Stands for "Gravatar version" and is used for cache busting.
       bindings:
         http:
-          $ref: '#/components/messageBindings/streamingHeaders'
+          headers:
```
Comment (derberg) [inline examples/gitter-streaming-asyncapi.yml:151]: $ref no supported in http header

File: examples/gitter-streaming-asyncapi.yml [hunk 2]
```diff
@@ -159,10 +167,12 @@ components:
             - "\r\n"
       bindings:
         http:
-          $ref: '#/components/messageBindings/streamingHeaders'
```
Comment (derberg) [inline examples/gitter-streaming-asyncapi.yml:162]: ref not supported in header + it must be a json schema

File: examples/operation-security-asyncapi.yml [hunk 3]
```diff
@@ -28,15 +28,8 @@ operations:
           - 'subscribe:auth_revocations'
     bindings:
       http:
-        type: request
```
Comment (derberg) [inline examples/operation-security-asyncapi.yml:31]: not supported in operation anymore, same headers

File: examples/operation-security-asyncapi.yml [hunk 4]
```diff
@@ -48,6 +41,10 @@ components:
           X-SIGNATURE:
             description: ECC message signature
             type: string
+          Content-Type:
```
Comment (derberg) [inline examples/operation-security-asyncapi.yml:44]: header can be normally a part of the message

File: examples/rpc-client-asyncapi.yml [hunk 5]
```diff
@@ -33,11 +33,7 @@ channels:
   rpc_queue:
     address: rpc_queue
     messages:
-      requestSum.message:
-        bindings:
-          amqp:
-            replyTo:
```
Comment (derberg) [inline examples/rpc-client-asyncapi.yml:39]: not supported by binding

File: examples/rpc-server-asyncapi.yml [hunk 6]
```diff
@@ -33,11 +33,7 @@ channels:
   rpc_queue:
     address: rpc_queue
     messages:
-      sum.message:
-        bindings:
-          amqp:
-            replyTo:
```
Comment (derberg) [inline examples/rpc-server-asyncapi.yml:39]: not supported by binding

File: examples/mercure-asyncapi.yml [hunk 7]
```diff
@@ -10,30 +10,29 @@ servers:
...
-      subscribe.message:
+      book:
         $ref: '#/components/messages/book'
     description: >-
       Every time a resource of type `http://schema.org/Book` is created or
       modified, a JSON-LD representation of the new version of this resource
       must be pushed in this Mercure topic.
     parameters:
-      id: {}
+      id:
```
Comment (derberg) [inline examples/mercure-asyncapi.yml:23]: this one was converted correct, as type was integer and no more info, even description provided, so result is basically an empty id parameter. But yeah, for best practice better share that description is a minimum

File: examples/social-media/backend/asyncapi.yaml [hunk 8]
```diff
@@ -19,64 +19,65 @@ servers:
...
     description: >-
       When an event from the broker arrives telling us to update the comment
       likes count on the frontend.
     servers:
       - $ref: '#/servers/mosquitto'
     parameters:
-      commentId: {}
-  update/comment/likes:
+      commentId:
+        $ref: '../common/parameters.yaml#/commentId'
```
Comment (derberg) [inline examples/social-media/backend/asyncapi.yaml:50]: since it is no longer schema it is better to promote also a separate file per parameters

**Review by dalelane — APPROVED** [review]
LGTM 👍

**Review by fmvilas — APPROVED** [review]

**Pull request merged**